### DISCUSSION TITLE
video 2

### DISCUSSION THREAD

**Discussion by JasonWang1125** [mrdbourke/pytorch-deep-learning#161] [General]
![image](https://user-images.githubusercontent.com/77429523/198889089-2ee7e168-1b35-40f4-be2a-5cb7302476be.png)

These red points inside the yellow line are curious, they are predicted by the model we trained, I think they should be blue

**Comment by fivefishstudios** [comment]
No, those are test data and actually are red. Meaning, the model isn't 100% accurate yet because the model forecasted them as blue when actually they are red. So the boundary in that section is still not perfect.

**Reply by JasonWang1125** [reply]
Thanks， I got it.

**Reply by mrdbourke** [reply]
@fivefishstudios good answer! Thank you for helping :)
